M2/Macaulay2/packages/Python.m2

```diff
@@ -26,10 +26,13 @@ rs = s -> (
      if debugLevel > 0 then stderr << "--python command: " << s << endl; 
      runString s);
 
-numContexts := 0
-nextContext := () -> (
-     numContexts = numContexts + 1;
-     "context" | toString numContexts)
+numContexts = 0
```

Comment:
What's this all about?  This package is for using when python is compiled in with M2, which it never is any more.

Reply:
This is a small change that was done when working with the Sage interface group. 
We do want M2 to have python interface. Don't we have `--with-python` anymore?

Reply:
By default, we build M2 without python.  I haven't tried it since 2009.  But if you want it for something, that's okay!